helper/helper_test.go

```diff
@@ -98,4 +189,60 @@ var _ = Describe("helper tests", func() {
 			Expect(isSubset).To(Equal(false))
 		})
 	})
+
+	Describe("doRestoreAgent Mocked unit tests", func() {
+		It("Test skip file in doRestoreAgent", func() {
+			Expect(1).To(Equal(1))
+
+			save_singleDataFile := *singleDataFile
+			save_content := *content
+			save_oidFile := *oidFile
+			save_isResizeRestore := *isResizeRestore
+			save_origSize := *origSize
+			save_destSize := *destSize
+			save_pipeFile := *pipeFile
+			save_onErrorContinue := *onErrorContinue
+
+			*singleDataFile = false
+			*content = 1
+			*oidFile = "testoid.dat"
+			*isResizeRestore = true
+			*origSize = 5
+			*destSize = 3
+			*pipeFile = "mock"
+			*onErrorContinue = true
```

Comment:
I think it is better to move the shenanigans with globals into  `BeforeEach()` and `AfterEach()`.
We can simply reset them into defaults without storing prev values.